.gitignore

```diff
@@ -1 +1,2 @@
-node_modules/
\ No newline at end of file
+node_modules/
+.idea
```

Comment:
What's .idea?

Reply:
It's a project folder generated by the WebStorm IDE.

Reply:
Cool. Add this to a global .gitignore, maybe? Bad habit to add this because it adds code that is only relevant to you.